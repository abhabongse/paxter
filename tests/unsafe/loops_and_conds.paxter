@python##"
    is_odd = lambda v: v % 2 == 1
"##\
@for[v in @|range(10)|]{\
    @v, @|v + 1|, @|v + 2|
    v is @if[@is_odd[@v]]{odd}|@if[not @is_odd[@v]]{even}
    v is @if[@is_odd[@v] then @"odd" else @"even"]
}\
