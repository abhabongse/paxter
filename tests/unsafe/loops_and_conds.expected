    0, 1, 2
    v is |even
    v is even
    1, 2, 3
    v is odd|
    v is odd
    2, 3, 4
    v is |even
    v is even
    3, 4, 5
    v is odd|
    v is odd
    4, 5, 6
    v is |even
    v is even
    5, 6, 7
    v is odd|
    v is odd
    6, 7, 8
    v is |even
    v is even
    7, 8, 9
    v is odd|
    v is odd
    8, 9, 10
    v is |even
    v is even
    9, 10, 11
    v is odd|
    v is odd
